modules/installation-configuration-parameters.adoc

```diff
@@ -309,6 +309,14 @@ ifdef::aws[]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.aws.amiID`
+|The The AWS AMI used to boot compute machines for the cluster. This is required for regions that require a custom RCOS AMI
```

Comment:
The field is not technically required, as there are other ways to specify the AMI to use. I see that this is the language that we use in more recent docs, too. I think it is confusing to state that all of the following are required.
`compute.platform.aws.amiID`
`controlPlane.platform.aws.amiID`
`platform.aws.amiID`
And there is no mention that the `platform.aws.defaultMachinePlatform.amiID` can be used instead of any of the preceding.
In these docs, we should at least add the section about the `platform.aws.amiID` field so that it is consistent with the docs for more recent OpenShfit versions. I see that Yunfei already mentioned this.
```suggestion
|The AWS AMI used to boot compute machines for the cluster. An AMI must be specified for regions that do not have a default RHCOS AMI.
```